src/components/JsonView.vue

```diff
@@ -35,10 +42,29 @@
   whitespace-pre
   break-normal
   rounded
-  text-left
-  p-4
-  text-slate-50
-  bg-slate-700
+  text-left;
+}
+
+.json-view__lines { @apply
+  py-1;
+  counter-reset: line;
+}
+
+.json-view__line { @apply
+  h-5
+  w-full
+  block
+}
+
+.json-view--with-line-numbers .json-view__line:before { @apply
+  inline-block
+  px-3
+  mr-5
+  border-r-[1px]
+  border-slate-200
```

Comment:
```suggestion
  border-slate-400
```
Can we make the number border dimmer? It's a tad bright atm (not sure if 400 is the way to go)